pkg/installer/installer.go

```diff
@@ -45,27 +53,36 @@ const (
 	UninstallSkipEtcdConfig       = "spec.uninstall.skipEtcd"
 
 	// dir and file names for in memory fs
-	etcdDir           = "etcd"
-	stosDir           = "storageos"
-	operatorDir       = "operator"
-	clusterDir        = "cluster"
-	stosOperatorFile  = "storageos-operator.yaml"
-	stosClusterFile   = "storageos-cluster.yaml"
-	etcdOperatorFile  = "etcd-operator.yaml"
-	etcdClusterFile   = "etcd-cluster.yaml"
-	kustomizationFile = "kustomization.yaml"
+	etcdDir              = "etcd"
+	stosDir              = "storageos"
+	operatorDir          = "operator"
+	clusterDir           = "cluster"
+	stosOperatorFile     = "storageos-operator.yaml"
+	stosClusterFile      = "storageos-cluster.yaml"
+	stosSecretsFile      = "storageos-secrets.yaml"
+	csiSecretsFile       = "storageos-csi-secrets.yaml"
+	stosStorageClassFile = "storageos-storageclass.yaml"
+	etcdOperatorFile     = "etcd-operator.yaml"
+	etcdClusterFile      = "etcd-cluster.yaml"
+	kustomizationFile    = "kustomization.yaml"
+	kubeDir              = "/root/.kube"
```

Comment:
Is it possible to use `$HOME` environment variable or some go func to determine home directory?